config/upload-api.yml

```diff
@@ -364,6 +364,50 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+    get:
+      summary: Get status of validation event on a file
```

Comment:
The wording of this is a little confusing: "status of validation event".
"Status of event" seems strange.  Events happen at a single moment in time.  They don't change status.
Is this actually "status of validation of file"?

Reply:
Thanks. Will change